### DISCUSSION TITLE
The new zopen package manager is now officially available!

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#388] [Announcements]
Hello everyone,

We are happy to announce that the new zopen package manager is now officially available! You can find download instructions [here](https://github.com/ZOSOpenTools/meta/releases).

Here's a quick overview of what you can expect from this official release:
* Centralized location for all your tools and libraries.
  * Customizable installation paths
  * Centralized zopen-config for streamlined management of tools
* Introduction of new zopen sub-commands: remove, upgrade, alt, clean, log, search, and query.
* Support for installing multiple versions of a given tool, with dynamic switching between versions using the alt sub-command.
* Removal of the requirement for Github OAUTH tokens.
* Improved zopen performance and enhanced .env processing.
* An option to install from Dev or Stable repositories.
* zopen framework now supports OMVS
* Disk space prompt before installation

We want to thank you for your valuable feedback during the beta phase! And a big thank you to @DevonianTeuchter for the amazing contributions towards this effort!

For a list of known issues, please visit our [issue tracker](https://github.com/ZOSOpenTools/meta/issues). If you encounter any new issues, don't hesitate to [report them](https://github.com/ZOSOpenTools/meta/issues/new).

We acknowledge that there may be a number of issues along the way during your migration and we appreciate your understanding as we continue to address any issues

Thank you!

**Comment by MikeFultonDev** [comment]
I did a [short video](https://ibm.box.com/s/xh9a6tld8z51a5x3qauhnk0idp7vgkrw) to show off some of the features:

**Comment by gngrossi** [comment]
v0.7.0 - the new zopen package manager

It looks like our z/OS system doesn’t have access to the URL (domain).
***ERROR: Failed to obtain json cache timestamp from https://zosopentools.github.io/meta/api/zopen_releases.json
***ERROR: Unable to install curl, jq and/or meta; see previous errors and retry the initilisation using the '--re-init' parameter

Our Z systems have access to github.com
Is it possible to use github.com for the json files?

thanks
